tools/wake-format/emitter.cpp

```diff
@@ -462,13 +591,13 @@ wcl::doc Emitter::walk_binary(ctx_t ctx, CSTElement node) {
 
   MEMO_RET(fmt()
                .walk(is_expression, WALK_NODE)
-               .consume_wsnl()
-               .space()
+               .consume_wsnlc()
+               .space()  // check for NL
```

Comment:
What's happening here?

Reply:
Ah these are just TODOs to fix these formatting cases since they'll be wrong if the sub format grows a NL. I wanted to have that fix be a dedicated PR though. I'll add TODOs